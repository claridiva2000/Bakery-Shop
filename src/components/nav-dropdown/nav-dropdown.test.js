import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

import NavDropdown from './nav-dropdown.component';

describe('NavDropdown component', () => {
  let wrapper;

  it('should render NavDropdown component', () => {
    wrapper = renderer
      .create(
        // Added MemoryRouter Component to test Link Components from react-router-dom
        <MemoryRouter>
          <NavDropdown />
        </MemoryRouter>
      )
      .toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
