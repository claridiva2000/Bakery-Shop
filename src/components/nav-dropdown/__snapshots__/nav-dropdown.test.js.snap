// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavDropdown component should render NavDropdown component correctly 1`] = `
<div
  className="nav-mobile"
>
  <div
    className="close"
  >
    <svg
      className="closeSvg"
      onClick={[Function]}
    >
      close.svg
    </svg>
  </div>
  <div
    className="links"
  >
    <div>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        href="/shop"
        onClick={[Function]}
      >
        Shop
      </a>
      <a
        href="/cart"
        onClick={[Function]}
      >
        Cart
      </a>
      <a
        href="/about"
        onClick={[Function]}
      >
        About Us
      </a>
      <a
        href="/contact"
        onClick={[Function]}
      >
        Contact Us
      </a>
    </div>
    <div
      className="button"
    >
      <button
        className="custom-button sign-in-with-Google"
      >
        Sign in with Google
      </button>
    </div>
  </div>
</div>
`;
